docs/training-apis/api-core/metrics.txt

```diff
@@ -0,0 +1,56 @@
+.. _core-metrics:
+
+################
+ Report Metrics
+################
+
+The Core API makes it easy to report training and validation metrics to the master during training
+with only a few new lines of code.
+
+For this example, create a new training script called ``1_metrics.py`` by copying the ``0_start.py``
+script from :ref:`core-getting-started`.
+
+Begin by importing import the ``determined`` module:
+
+.. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
+   :language: python
+   :start-after: NEW: import determined
+   :end-before: def main
+
+In your ``if __name__ == "__main__"`` block, wrap the entire execution of ``main()`` within the
+scope of :meth:`determined.core.init`, which prepares resources for training and clean them up
+afterwards. Add the ``core_context`` as a new argument to ``main()`` because the Core API is
+accessed through the ``core_context`` object.
+
+.. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
+   :language: python
+   :start-at: if __name__ == "__main__"
+
+Within ``main()``, add two calls. One to report training metrics, which is called periodically
+during training, and one to report validation metrics, which is called every time a validation runs.
+The model is so simple that there is no real concept of validation, but imagine you did one
```

Comment:
This sentence is too informal. Calling on the 'imagination' is weird. At the very least, change "finished" to "completes"